app/views/submissions/_form.html.erb

```diff
@@ -0,0 +1,46 @@
+
+<%= bootstrap_form_for [@project, @submission] do |form| %>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        New Job Submission
+      </div>
+
+      <div class="panel-body", role="main">
+        <%= form.text_field :name, required: true %> 
+        <%= form.text_field :frames, required: true %>
+        <%= form.text_field :camera, required: true %> 
+        <%= form.text_field :file, required: true, 
+          'data-filepicker': @project.directory %>
+        
+        
+        <%= form.select :cluster, OODClusters.map { |cluster| [ "#{cluster.metadata.title}", cluster.id ] }, 
+            { label: "Cluster:", :include_blank => true }, { class: "selectpicker", id: "batch_host_select" } %>
+
+
+        <button type="button" name="show_extra_vars" 
```

Comment:
For progressive disclosure controls in forms use https://getbootstrap.com/docs/3.3/javascript/#collapse-example-accordion instead of a custom implementation.